core/lib/libmbedtls/include/mbedtls_config.h

```diff
@@ -0,0 +1,55 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2017, ARM Limited
+ */
+
+#ifndef MBEDTLS_FEATURE_CONFIG_H
+#define MBEDTLS_FEATURE_CONFIG_H
+
+#include "mbedtls/config.h"
```

Comment:
I think it would be better if this file was self-contained and only defined what's needed instead of defining "everything" and then only remove what causes problems.

This way of doing it also seems to miss taking advantage of the logic provided by `"check_config.h"`.

Reply:
I want to reuse the `config.h`, so that we do not to maintain it. Because the `config.h` may be updated for bug fix or new feature. 
And also, sorry for not think about the `check_config.h`. 
What about include the `check_config.h` again in `mbedtls_config.h`?

Reply:
MbedTLS is compiled here for a purpose, to provide features needed for the crypto API. So even if there's new features it's quite likely that they are not needed. By only disabling what's causing problems we're not really in control over the configuration. I'd prefer if we started at the other end, only enabling what's needed and nothing more. That should result in a fairly small config file.

Reply:
OK, I agree with you. I will try to only enable what we need and it will also decrease "bin" size.

Reply:
Thanks